### PR TITLE
adding filebeat PaaS Logs

### PR DESCRIPTION
**Pull request by jehuty0shift**

### CODE REVIEW

File: en/_posts/2016-02-25-filebeat-logs.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    # Number of rotated log files to keep. Oldest files will be deleted first.
+    keepfiles: 7
+  # Sets log level. The default log level is error.
+  # Available log levels are: critical, error, warning, info, debug
+level: info
+``` 
+
+####SSL CA Certificate 
+```bash
+-----BEGIN CERTIFICATE-----
```
Comment (yadutaf) [inline en/_posts/2016-02-25-filebeat-logs.md]: Can you make it available for download somewhere? That would make automation even easier. Thanks.

**Comment by yadutaf** [comment]
Can you rebase and squash your commits ? Thanks.

File: en/_posts/2016-02-25-filebeat-logs.md (hunk 1 above)
Comment (jehuty0shift) [inline en/_posts/2016-02-25-filebeat-logs.md]: No problem, where should I put it ? Do we have any specific location for files like this. We could in fact upload some exemple files or code snippets.

**Comment by jehuty0shift** [comment]
I did it.

File: en/_posts/2016-02-25-filebeat-logs.md (hunk 1 above)
Comment (yadutaf) [inline en/_posts/2016-02-25-filebeat-logs.md]: I'd put it on a PaaS log domain.

**Comment by yadutaf** [comment]
:+1: LGTM

**Pull request merged**